requirements.txt

```diff
@@ -0,0 +1,3 @@
+ops==1.0.1
+pymssql==2.1.5
+git+https://github.com/juju/charm-helpers.git@09208c2#egg=charmhelpers
```

Comment:
pinning to versions (and individual commits, in the last case) like this seems inadvisable to me, given people not developing the charm would typically install it from the store (where you'd push a working charm, i.e. the versions are fixed in the charm); pinning will mean you'll need to remember to bump these all the time.

Reply:
I agree. I'll remove the pins for all the `requirements.txt` deps.